src/dbally/iql_generator/iql_prompt_template.py

```diff
@@ -60,3 +75,27 @@ def _validate_iql_response(llm_response: str) -> str:
     ),
     llm_response_parser=_validate_iql_response,
 )
+
+default_few_shot_iql_template = IQLFewShotPromptTemplate(
+    chat=(
+        {
+            "role": "system",
+            "content": "You have access to API that lets you query a database:\n"
+            "\n{filters}\n"
+            "Please suggest which one(s) to call and how they should be joined with logic operators (AND, OR, NOT).\n"
+            "Remember! Don't give any comments, just the function calls.\n"
+            "The output will look like this:\n"
+            'filter1("arg1") AND (NOT filter2(120) OR filter3(True))\n'
+            "DO NOT INCLUDE arguments names in your response. Only the values.\n"
+            "You MUST use only these methods:\n"
+            "\n{filters}\n"
+            "It is VERY IMPORTANT not to use methods other than those listed above."
+            "Here's a list of examples:\n"
+            "\n{examples}\n"
```

Comment:
hmm maybe we can put few-shots as a part of conversation, then we wouldn't have to create separate prompt for that.

for example:
```
    chat=(
        {
            "role": "system",
            "content": "You have access to API that lets you query a database:\n"
                            .......
        },
        {"role": "user", "content": "<QUESTION 1>"},
        {"role": "assistant", "content": "<ANSWER 1>"},
        {"role": "user", "content": "<QUESTION 2>"},
        {"role": "assistant", "content": "<ANSWER 2>"},
        {"role": "user", "content": "{question}"},
    ),
```